test/reconstruction/CoreModel.jl

```diff
@@ -1,51 +1,3 @@
-@testset "Find exchange reactions and metabolites" begin
```

Comment:
this got completely replaced?

Reply:
ah no, it got moved to `test/utils/looks_like.jl`